numpy/doc/creation.py

```diff
@@ -44,10 +43,10 @@
 NumPy has built-in functions for creating arrays from scratch:
 
 zeros(shape) will create an array filled with 0 values with the specified
-shape. The default dtype is float64.
+shape. The default dtype is float64. ::
 
-``>>> np.zeros((2, 3))
-array([[ 0., 0., 0.], [ 0., 0., 0.]])``
+ >>> np.zeros((2, 3))
+ array([[ 0., 0., 0.], [ 0., 0., 0.]])
```

Comment:
Indeed, the page is currently rendered improperly http://www.numpy.org/devdocs/user/basics.creation.html#intrinsic-numpy-array-creation

Reply:
I think normally we'd indent this four spaces, not one

Reply:
Although what you have is consistent with the rest of the file, so nevermind